hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release adds the strategy :func:`~hypothesis.extra.numpy.valid_tuple_axes`,
+which generates tuples of axis-indices that can be passed to the ``axis`` argument
+in NumPy's sequential functions (e.g. ``numpy.sum``).
```

Comment:
👍

Reply:
Regarding the release notes, should I mention the `none() | valid_tuple_axes()` trick here in addition to the docstring? It seems like it might be something worth advertising.